File: Javascript/Data Structure/largestinarray.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter numbers separated by space: ', function(input) {
    const arr = input.split(' ').map(Number);
    const max = Math.max(...arr);
    console.log('Largest number:', max);
    rl.close();
});

File: Java Practise/LargestInArray.java
public class LargestInArray {
    public static void main(String[] args) {
        int[] arr = {10, 45, 2, 99, 23};
        int max = arr[0];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > max)
                max = arr[i];
        }
        System.out.println("Largest Element: " + max);
    }
}
